queries/typespec/highlights.scm

```diff
@@ -0,0 +1,125 @@
+(identifier_or_member_expression) @type
```

Comment:
`@variable` / `@variable.member` ?

Reply:
Semantically, these are always types, since typespec is a definition language for APIs. Since there aren't any variable being defined, any identifier we can refer to is pretty much a type, or a type within a namespace in the case of a member expression. As an example:

```
model User { ... }

namespace Blog {
  model Comment { ... }
}

model Post {
  author: User // <- This is an identifier
  comments: Blog.Comment[] // <- This is a member expression
}
```